source/node/introduction.rst

```diff
@@ -41,18 +41,21 @@ ethereum keystore.
 Session
 ^^^^^^^
 
-Provide an environment for the user access his ID provider, key manager and
-manage all the channels, throughout their lifecycle. In the context of a
-session, he/she can open a state channel, do some transactions through channels
-and settle the channel. Each session runs its own instance of state channel
-network client, ID provider, key manager; and hence provides a complete
-isolated environment within the node. It is this feature that enables multiple
-users to use the same node by having dedicated sessions for each of them.
+Provide an environment for the user to access his ID provider, key manager and
+all the channels opened within this session. In the context of a session,
+he/she can open a state channel, do some transactions on the channels and
+settle it. Each session runs its own instance of state channel network client,
+ID provider, key manager; and hence provides a complete isolated environment
+within the node. It is this feature that enables multiple users to use the same
+node by having dedicated sessions for each of them.
 
 User need not be worried about losing his/her data when a sudden closure of
-session occurs due to some network error or because of some other reasons.  All
+session occurs due to some network error or because of some other reasons. All
 the data within the sessions are continuously persisted. User can reopen the
-session at any time. User can close the session according to his/her needs. 
+session at any time and the last known state of the session will be restored.
+User also has the option to close a session that has open channels by using
+``force`` option. In this case as well, the session data is persisted and
+restored when the session is reopened.
```

Comment:
```suggestion
restored.
```
Or we can say these can be retrieved when session is reopened.

Reply:
I think in this case, `restored` is more appropriate than retrieved.

The phrase `when the session is reopened` also seems necessary.